src/Microsoft.IdentityModel.Tokens.Saml/Saml/Exceptions/SamlValidationError.cs

```diff
@@ -4,11 +4,18 @@
 using System;
 using System.Diagnostics;
 
+#nullable enable
```

Comment:
This is because the inner exception might be null now because we didn't throw the exception whereas before it wouldn't?

Reply:
We are enabling nullability checks on all of the new files to ensure we annotate possible nulls and do all the checks required.
Ideally one day, the whole project would have nullability annotations and would follow the best practices, but for now we are applying this to the new code on this workstream.